test/e2e/page-objects/pages/notifications/first-time-turn-on-modal.ts

```diff
@@ -0,0 +1,31 @@
+import { Driver } from '../../../webdriver/driver';
+
+class FirstTimeTurnOnNotificationsModal {
+  private driver: Driver;
+
+  private readonly turnOnButton =
+    '[data-testid="turn-on-notifications-button"]';
+
+  constructor(driver: Driver) {
+    this.driver = driver;
+  }
+
+  async check_pageIsLoaded(): Promise<void> {
+    try {
+      await this.driver.waitForSelector(this.turnOnButton);
+    } catch (e) {
+      console.log(
+        'Timeout while waiting for TurnOnNotificationsModal to be loaded',
+        e,
+      );
+      throw e;
+    }
+    console.log('TurnOnNotificationsModal is loaded');
+  }
+
+  async clickTurnOnNotifications(): Promise<void> {
+    await this.driver.clickElement(this.turnOnButton);
```

Comment:
use `clickElementAndWaitToDisappear` to make sure the modal closes, that will reduce flakiness in the test execution